youtube_dl/extractor/qqmusic.py

```diff
@@ -58,6 +58,11 @@ def m_r_get_ruin():
         curMs = int(time.time() * 1000) % 1000
         return int(round(random.random() * 2147483647) * curMs % 1E10)
 
+    def _filter_lrc(self, data):
+        lyrics_expr = r'(\[[0-9]{2}:[0-9]{2}\.[0-9]{2,}\])([^\n]+)'
+        texts = re.findall(lyrics_expr, data)
+        return ''.join("%s%s\n" % (time_stamp, text) for time_stamp, text in texts)
```

Comment:
.lrc may contain another tags apart from time tags (ID tags) that are completely filtered out now.